### PR TITLE
[Feature][UI Next] Add project task instance.

### PR DESCRIPTION
**Pull request by songjianet**
## Purpose of the pull request
[#7982](https://github.com/apache/dolphinscheduler/issues/7928)

image
image
image

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by labbomb — DISMISSED** [review]
LGTM

File: dolphinscheduler-ui-next/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "echarts": "^5.2.2",
     "lodash": "^4.17.21",
     "monaco-editor": "^0.31.1",
-    "naive-ui": "2.23.2",
+    "naive-ui": "2.24.6",
```
Comment (CalvinKirs) [inline dolphinscheduler-ui-next/package.json:20]: Newly introduced dependencies?

Comment (songjianet) [reply dolphinscheduler-ui-next/package.json:20]: No, it's a dependency upgrade

Comment (CalvinKirs) [reply dolphinscheduler-ui-next/package.json:20]: Looks like a version update. You may need to check the corresponding license

Comment (songjianet) [reply dolphinscheduler-ui-next/package.json:20]: license has not changed

Comment (songjianet) [reply dolphinscheduler-ui-next/package.json:20]: After the new ui development is basically completed, I will deal with the license

**Review by zhongjiajie — APPROVED** [review]
LGTM

File: dolphinscheduler-ui-next/package.json (hunk 1 above)
Comment (zhongjiajie) [reply dolphinscheduler-ui-next/package.json:20]: 👍 Great, thanks @CalvinKirs point it out

**Pull request merged**